components/Search/Hit.tsx

```diff
@@ -50,11 +51,24 @@ const TagsContainer = styled.div`
   gap: ${tokens.spacings.comfortable.small};
 `
 
+const TopContainer = styled.div`
+  margin-top: ${tokens.spacings.comfortable.medium};
```

Comment:
where did you get this margin from? from the screen shot your shared in slack, it seems like only cards with a "new" tag get a lot of top margin. i suspect this line is the culprit, and i think it can be removed